### PR TITLE
Login signup

### PR DESCRIPTION
**Pull request by shruthipatil11**
Added login and sign up pages. Kindly provide your suggestions

### CODE REVIEW

File: src/app/auth/auth.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+grid-area: content;
+display: flex;
+flex-direction: column;
+justify-content: center;
+align-items: center;
+row-gap: 1rem;
+width: 60vw;
+margin: auto;
+}
+
```
Comment (NeerajVodala) [inline src/app/auth/auth.css]: use a formatter so that spacing will be consistent throughout the code.

**Review by NeerajVodala — APPROVED** [review]
Looks good to me.
All the best.

**Comment by SumitSaundadkar** [comment]
Good to go.

File: src/app/auth/auth.css (hunk 1 above)
Comment (shruthipatil11) [reply src/app/auth/auth.css]: done

**Review by SinchanaMS — APPROVED** [review]
The code looks neat! UI looks good too. Good work!

**Pull request merged**